SilasKenneth/fast-food-fast-ui
# [Finishes 160131185] Create the homepage for the application

#### What this pull request does
Creates the homepage for the fast-food-fast website
##### Description of task to be completed
 Create the homepage
    -  Add links to `login` and `signup`
    - The main page layout
##### Relevant pivotal tracker story
[#160131185](https://www.pivotaltracker.com/n/projects/2194200/stories/160131185)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder
  - Open the index.html file in your browser and that should work

#### Screenshot
![home](https://user-images.githubusercontent.com/11555354/44849936-5fdfa700-ac64-11e8-8d49-1f10893526be.png)
